ssr-opengraph/prettier.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import("prettier").Config} */
```

Comment:
If possible `.prettierrc` with `semi: false`? in a follow up?

Reply:
updated